db/seed.js

```diff
@@ -95,5 +95,12 @@ class dataGenerator {
 }
 
 let generator = new dataGenerator();
-let product = generator.generateProduct();
-productDetails.create(product);
+products = [];
+let product;
+for (let i = 0; i < 150; i++) {
+  product = generator.generateProduct();
+  products.push(product);
+}
+productDetails.create(products)
+  .then(() => mongoose.connection.close())
+  .catch((err) => console.log(err));
```

Comment:
Don't forget to add a new line at the end of every file ~ a linter tool would be helpful!